swan-lake/learn-the-platform/ballerina-persist/persist_client.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Persist Client API
+description: The sections gives details on the generated Persist Client API.
+keywords: ballerina, programming language, ballerina packages, persist, persist client, persist client api
+permalink: /learn/ballerina-persist/persist-client-api/
+active: persist_overview
+intro: Persist client is used to perform CRUD operations on the data source. Each entity type will have five resource methods for each operation. The resource methods are get, get(get by identity), post, put, and delete. Persist CLI tool will generate the derived Entity Types and the clients from the model definition.
```

Comment:
```suggestion
intro: Persist client is used to perform CRUD operations on the data source. Each entity type will have five resource methods for each operation. The resource methods are get, get(get by identity), post, put, and delete. Persist CLI tool will generate the derived Entity Types and the Clients from the model definition.
```

Reply:
Not sure about this one, but having `Entity Type` and `client`, seems inconsistant